src/components/Model.js

```diff
@@ -13,7 +15,9 @@ const customStyles = {
     transform: 'translate(-50%, -50%)'
   }
 };
+
 Modal.setAppElement('#root');
+
 class Model extends React.Component {
```

Comment:
I think you are going to remove this modal ... but seems like a typo here `Model`.